pkg/util/cmd/encoder.go

```diff
@@ -45,10 +45,14 @@ func NewTiDBEncoder(cfg zapcore.EncoderConfig) zapcore.Encoder {
 	return &tidbEncoder{cfg, 0, _pool.Get()}
 }
 
-func (c tidbEncoder) Clone() zapcore.Encoder {
+func (c tidbEncoder) clone() *tidbEncoder {
 	return &tidbEncoder{c.EncoderConfig, 0, _pool.Get()}
 }
 
+func (c tidbEncoder) Clone() zapcore.Encoder {
+	return c.Clone()
```

Comment:
```suggestion
	return c.clone()
```
Are you sure it calls itself?

Reply:
I should have changed this to `c.clone()`. Maybe I made mistakes in `--rebase`.